tenderly-suite/token-transfers/token-transfers/handleTokenTransfer.js

```diff
@@ -0,0 +1,184 @@
+const { ethers } = require('ethers');
+const {
+    utils: { toUtf8Bytes },
+} = ethers;
+const axios = require('axios').default;
+
+const ERC20_ABI = ['function symbol() external view returns (string)', 'function decimals() external view returns (uint8)'];
+const ITS_ABI = ['function validTokenAddress(bytes32 tokenId) external view returns (address)'];
```

Comment:
Might eventually want to move ABIs to separate files if you plan on calling more functions within these contracts and reusing them across different monitoring scripts